install_config/cluster_metrics.adoc

```diff
@@ -439,6 +439,12 @@ pods are placed onto nodes with specific labels. For example,
 remove the HOSA from an installation. HOSA can be used to collect custom
 metrics from your pods. This component is currently in
 Technology Preview and is not installed by default.
+
+|`openshift_metrics_hawkular_hostname`
+|Set `openshift_metrics_hawkular_hostname` when executing the
```

Comment:
Maybe just drop the name in the description and just say "Set when executing..."

Reply:
Done, thanks! :dog: